ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/EeaSendTransactionJsonParameters.java

```diff
@@ -146,7 +147,7 @@ public static EeaSendTransactionJsonParameters from(final JsonRpcRequest request
   }
 
   private void validateValue(final String value) {
-    if (!decodeQuantity(value).equals(BigInteger.ZERO)) {
+    if (value != null && !decodeQuantity(value).equals(BigInteger.ZERO)) {
```

Comment:
what if value is null? `if (value == null || !decodeQuantity(....))` might be more appropriate?

Reply:
I actually want to skip the validation if it's null. The null check shouldn't be in this method, I've moved it in the value setter where I think it belongs.